code/controllers/subsystems/garbage.dm

```diff
@@ -393,9 +393,14 @@ SUBSYSTEM_DEF(garbage)
 	set name = "Find References"
 	set src in world
 
-	find_references(FALSE)
+	user_find_references()
 
-/datum/proc/find_references(skip_alert)
+/datum/proc/user_find_references()
+	if(alert("Running this will lock everything up for about 5 minutes.  Would you like to begin the search?", "Find References", "Yes", "No")) == "No")
```

Comment:
General view on alerts is to lead with "No" so you can't accidentally hit yes by having the alert pop up and take focus.

Reply:
Just moving it from the other proc, but I guess I can change it anyway.